di_website/workforus/models.py

```diff
@@ -0,0 +1,108 @@
+from django.db import models
+
+from wagtail.core.models import Page
+from wagtail.core.fields import StreamField, RichTextField
+from wagtail.core import blocks
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    InlinePanel,
+    MultiFieldPanel,
+    PageChooserPanel,
+    StreamFieldPanel
+)
+from wagtail.images.blocks import ImageChooserBlock
+from wagtail.snippets.models import register_snippet
+
+from di_website.common.base import hero_panels
+from di_website.common.mixins import BaseStreamBodyMixin, HeroMixin
+from di_website.vacancies.models import VacancyPage
+from .blocks import BenefitsStreamBlock, TeamStoryStreamBlock
+
+from modelcluster.fields import ParentalKey
+
+
+@register_snippet
+class Values(models.Model):
```

Comment:
The value block has a "self" variable that is difficult to load from snippets; 

It throws an error `push() got multiple values for argument 'self'` and crashes the page

Instead of creating another include, since the values text is small i've added it to the work for us page.